usr/src/uts/armv8/io/arm_gtmr.c

```diff
@@ -378,9 +378,9 @@ arm_gtmr_attach(dev_info_t *dip, ddi_attach_cmd_t cmd)
 	 * - 2: virtual
 	 * - 3: hypervisor
 	 *
-	 * If booted at EL1 we use the virtual timer (index 2), otherwise we
-	 * use the non-secure physical timer (index 1). We only ever use one
-	 * of the offered interrupts.
+	 * If booted at EL2 we use the hypervisor timer (index 2), otherwise
+	 * we use the virtual timer (index 2). We only ever use one of the
```

Comment:
they're not both at index 2, right?

Reply:
Ugh, I revised the other commit but didn't even look at this one. Fixed.